projects/plugins/jetpack/_inc/lib/tonesque.php

```diff
@@ -38,6 +38,8 @@ class Tonesque {
 		 * @param string $image_url Image URL.
 		 */
 		public function __construct( $image_url ) {
+			_deprecated_function( 'Tonesque::__construct', 'jetpack-$$next-version$$' );
```

Comment:
```suggestion
			_deprecated_function( __METHOD__, 'jetpack-$$next-version$$' );
```
Like with https://github.com/Automattic/jetpack/pull/38338/files#r1720052504